README.md

```diff
@@ -95,7 +95,7 @@ Kong can easily be provisioned to Kubernetes cluster using the following steps:
 
 3. **Using Kong:**
 
-    Quickly learn how to use Kong with the [5-minute Quickstart](/docs/latest/getting-started/quickstart).
+    Quickly learn how to use Kong with the [5-minute Quickstart](https://getkong.org/docs/0.9.x/getting-started/quickstart/).
```

Comment:
I think the 0.9.x path segment should be replaced by `latest` to ensure this links to the current relevant getting started guide :)